pages/en/android/optimizing.md

```diff
@@ -19,7 +19,7 @@ The implementation of Android apps on Chrome OS includes basic multi-window supp
 
 Users can resize the window that contains your Android app, as shown in figure 1. To ensure that your free-form windows resize smoothly and are able to display their entire contents to the user, read the guidelines in [Window Management](/{{locale.code}}/android/window-management).
 
-#1[A resizable app window](/images/android/optimizing/resizable.png)
+#1[A resizable app window](/images/android/window-management/resizable.png)
```

Comment:
Should this stay in the optimizing folder since its on the optimizing.md file?

Reply:
Ah nevermind, I realize it's because the window management page also uses the same image again.